engine/src/main/scala/cromwell/jobstore/JobStoreReaderActor.scala

```diff
@@ -20,15 +21,24 @@ class JobStoreReaderActor(database: JobStore, override val serviceRegistryActor:
     with ActorLogging {
 
   override def processHead(head: CommandAndReplyTo[QueryJobCompletion]) = instrumentedProcess {
+    log.info("{}: Processing Job store check", head.command.jobKey.tag)
+
     val action = database.readJobResult(head.command.jobKey, head.command.taskOutputs)
+
     action onComplete {
-      case Success(Some(result)) => head.replyTo ! JobComplete(result)
-      case Success(None) => head.replyTo ! JobNotComplete
+      case Success(Some(result)) =>
+        head.replyTo ! JobComplete(result)
+      case Success(None) =>
+        head.replyTo ! JobNotComplete
```

Comment:
TOL I'd prefer not to introduce purely whitespace changes if we find ourselves reviewing this later under time constraints...

Reply:
I agree. I think I originally had different logging per case before I consolidated into the for comp. Will revert these changes.